### DISCUSSION TITLE
Welcome to ETHMerge.com Discussions!

### DISCUSSION THREAD

**Discussion by InsideTheSim** [InsideTheSim/ethmerge.com-content#17] [General]
## 👋 Welcome!
  I'm using Discussions as a place to connect with other members of the community. I hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

Let's make Ethereum look **good** together. Feel free to drop a line and introduce yourself in this thread. I'm happy to help newcomers get involved.

**Comment by Besiktas19033** [comment]
809fc7afaffaad7d31c9b3761c0f35226098da59
